jobs/haproxy/spec

```diff
@@ -114,3 +118,28 @@ properties:
           - 10.20.10.11
           backend_port: 80 # optional - sets backend port - otherwise defaults to `port`
           ssl: true        # optional - enables ssl, and uses the `ha_proxy.ssl_pem` provided key
+
+  keepalived.vip:
+    description: VIP that will be given to master
+  keepalived.healthcheck_name:
+    description: name of healthcheck that will be performed
+    default: check_haproxy
+  keepalived.healthcheck_command:
+    description: check that will be performed
+    default: killall -0 haproxy
+  keepalived.interface:
+    description: interface keepalived will use to mount the VIP
+    default: eth0
+  keepalived.masterip:
+    description: IP of masternode
+  keepalived.vrrp_instance_name:
+    description: name of the VRRP instance
+  keepalived.master_priority:
+    description: master weight
+    default: 101
+  keepalived.slave_priority:
+    description: slave weight
+    default: 100
+  keepalived.lvs_id:
+    description : Keepalived process identifier
```

Comment:
Good catch (I came to the same conclusion independently rethinking about it). 

It would be ideal to make the property required and without a default value, as to make it clear to operators they indeed need to specify a unique value per keepalived deployment per subnet. I'm not sure the current spec file syntax https://bosh.io/docs/jobs.html#spec supports enforcing mandatory properties, I guess the logic needs to be in the ERB template, and failing the job start with hopefully an error message in the job log (which is not very user friendly)

/CC @poblin-orange any suggestion as to make a bosh job property mandatory without default ?

If not, then we might prefer a default lvs_id value "haproxy-bosh", as to ease the 1st time newcomer ops experience.